pkgs/top-level/all-packages.nix

```diff
@@ -7144,7 +7144,7 @@ with pkgs;
 
   grabserial = callPackage ../development/tools/grabserial { };
 
-  mypy = callPackage ../development/tools/mypy { };
+  mypy = python3Packages.callPackage ../development/tools/mypy { };
```

Comment:
I can change it if you want, but I personally prefer not to rely on the structure of attributes in a package definition.